utils/src/main/scala/za/co/absa/enceladus/utils/fs/HadoopFsUtils.scala

```diff
@@ -31,10 +28,10 @@ import scala.util.Try
  * A set of functions to help with the date partitioning and version control
  */
 
-class HdfsUtils(conf: Configuration) extends DistributedFsUtils {
+class HadoopFsUtils()(implicit fs: FileSystem) extends DistributedFsUtils {
```

Comment:
I was trying to insinuate to the user/reader that this is not bound to the "standard" HDFS - the usually used distributed FS used with Spark, but backed by Hadoop FS API - which can turn out to be HDFS, S3, on anything else.

I know that the stack layering can be confusing, that is roughly what I was trying to demonstrate.

What is your option on this when you know my reasoning? Would you suggest keeping it as is, even though it may be a bit misleading?